.ci/benchmarks.groovy

```diff
@@ -109,3 +125,47 @@ def withESBenchmarkEnv(Closure body) {
     body()
   }
 }
+
+def downloadPNGReport() {
+  withSecretVault(
+      secret: "${KIBANA_SECRET}", 
+      data: ['username': 'KIBANA_USER', 
+            'password': 'KIBANA_PASSWORD', 
+            'endpoint': 'KIBANA_ENDPOINT'] ) {
+    sh(label: 'Run unit tests', script: ".ci/scripts/download-png-from-kibana.sh $KIBANA_ENDPOINT $KIBANA_USER $KIBANA_PASSWORD ${env.PNG_REPORT_FILE}")
```

Comment:
To play safe, I'd say those env variables are implicitly used within the script, to avoid any weirdness from the security point of view in the deprecated Blue Ocean view. See https://issues.jenkins.io/browse/JENKINS-59214, or use the single quotes as proposed in https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#interpolation-of-sensitive-environment-variables

```suggestion
    sh(label: 'Run unit tests', script: '.ci/scripts/download-png-from-kibana.sh $KIBANA_ENDPOINT $KIBANA_USER $KIBANA_PASSWORD $PNG_REPORT_FILE')
```

or


```suggestion
    sh(label: 'Run unit tests', script: '.ci/scripts/download-png-from-kibana.sh')
```

and change the `.ci/scripts/download-png-from-kibana.sh` to use those env variables